src/sass/app.scss

```diff
@@ -340,4 +340,9 @@ body body{
   & .arrow {
     left: calc(50% - 0.4rem);
   }
-}
\ No newline at end of file
+}
+
+// Hide Zendesk Chat until it is ready to be displayed
+.zopim {
```

Comment:
This makes it so the widget is hidden by default, until it is ready to be displayed. I found this necessary because the `setOnConnected` will display the widget for a brief moment before it figures out it needs to disappear.

Reply:
Had to abandon this change as there is some weirdness with if the user tries to dismiss the chat window. It's somewhat had to explain, but I can easily demo if anyone is interested.

So for now if the widget attempts to display erroneously on the page it might appear for a very brief moment before it dismisses itself.